core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1221 @@
+    width: 100%;
+  }
+  .oppia-about-page-section-five .oppia-about-classroom-button-container {
+    float: left;
+  }
+  .oppia-about-page-section-six {
+    background-color: white;
+    clear: both;
+    display: flex;
+    height: 70vh;
+    justify-content: center;
+    margin: 0 auto;
+    padding-right: 2vw;
+    padding-top: 6.5vh;
+    width: 100%;
+  }
+  .oppia-about-page-section-six .oppia-section-six {
+    align-content: center;
+    max-width: 1440px;
+    width: 1440px;
+  }
+  .oppia-about-page-section-six .oppia-about-section-six-right {
+    float: right;
+    padding-left: 2vw;
+    padding-right: 2vw;
+    width: 50%;
   }
-  .oppia-about-page-container .oppia-about-clearfix {
+  .oppia-about-page-section-six .oppia-about-outcome-title {
+    font-weight: normal;
+    margin: 0;
+  }
+  .oppia-about-page-section-six .oppia-about-outcome-content {
+    margin: 0;
+    padding-top: 4vh;
+  }
+  .oppia-about-page-section-six .oppia-about-outcome-title-mobile {
+    display: none;
+  }
+  .about-page .oppia-about-page-section-six {
+    background-color: white;
+    background-image: url(/assets/images/about/background_dsk_2.svg);
+    background-size: 100% 70vh, cover;
+  }
+  .oppia-about-page-section-six .oppia-about-book-image img {
+    float: left;
+    height: auto;
+    padding-left: 4vw;
+    width: 45%;
+  }
+  .oppia-about-section-six-right .oppia-about-classroom-button-container {
+    padding-top: 5vh;
+  }
+  .about-page .oppia-about-page-section-seven {
+    background-color: white;
     clear: both;
-    content: "";
-    display: table;
+    height: auto;
+    margin: 0 auto;
+    max-width: 100%;
+    padding-top: 5vh;
+    text-align: center;
+  }
+  .oppia-about-page-section-seven .oppia-get-started-title {
+    font-weight: 400;
+    margin: 0;
+    padding-bottom: 5vh;
+  }
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-one,
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-two,
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-three {
+    padding-bottom: 4vh;
   }
-  .oppia-about-page-container .oppia-about-otter {
-    width: 200px;
+  .oppia-about-page-section-four .oppia-about-button,
+  .oppia-about-page-section-five .oppia-about-button,
+  .oppia-about-page-section-six .oppia-about-button {
+    margin-top: 15px;
   }
-  .oppia-about-page-container .oppia-about-otter-container {
-    bottom: -2.3em;
-    display: inline-block;
-    position: absolute;
+  .about-page .oppia-about-button {
+    background-color: #015c53;
+    border-radius: 0;
+    color: #fff;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    margin-left: 0;
+    margin-top: -20px;
+    min-width: 250px;
+    padding: 10px;
+    text-transform: uppercase;
+    width: 220px;
   }
-  .oppia-about-page-container .oppia-about-tab-text {
-    margin-top: -10px;
+  /* stylelint-disable no-descending-specificity */
```

Comment:
Why are we disabling styleint here?

Reply:
It was done for a button we earlier had but we don't need it now